jobmanager/WorkerThread.h

```diff
@@ -10,18 +12,28 @@ class WorkerThread
 {
 public:
     explicit WorkerThread(jobmanager::JobManager& jobManager);
+    ~WorkerThread();
 
     void stop();
 
     static double getWaitTime(); // ms
     static double getWorkTime(); // ms
+
+    static bool isWorkerThread();
+    static bool yield();
+
 private:
     std::atomic<bool> _running;
     jobmanager::JobManager& _jobManager;
 
     void run();
+    bool processJobs();
     static void threadEntry(WorkerThread* instance);
+    uint32_t processYieldedJobs();
 
+    std::array<JobBase*, 512> _yieldedJobs;
```

Comment:
question how to dimension this. It could be by config and dynamically allocated as it is done at startup.
With multiple threads there can be a lot of yielded tasks.